packages/docs/src/styles/_sections.scss

```diff
@@ -69,6 +69,7 @@
 }
 
 #{section-name("Labels")},
+#{section-name("Helper Text")},
```

Comment:
using `line-height` breaks down as soon as the label overflows to a second line (or do we want to leave it to the consumer on how to handle? (whether it be with ellipsis etc?)